dotnet/AspNetMvc5Ef6/Global.asax.cs

```diff
@@ -45,5 +49,25 @@ protected void Application_End()
             // Close the Sentry SDK (flushes queued events to Sentry)
             _sentry?.Dispose();
         }
+
+        protected void Application_BeginRequest()
+        {
+            var method = Context.Request.HttpMethod;
+            var path = Context.Request.Path;
+
+            // Start a transaction that wraps the current request and bind it to the scope
+            var transaction = SentrySdk.StartTransaction(
+                $"{method} {path}",
+                "http.server"
+            );
+
+            SentrySdk.ConfigureScope(scope => scope.Transaction = transaction);
+        }
+
+        protected void Application_EndRequest()
+        {
+            // Finish the currently active span
+            SentrySdk.GetSpan()?.Finish();
```

Comment:
Best to add to the HttpContext, there's some data bag in it.
So we can finish the transaction by grabbing the reference from it